command/pip/in-streamlit-service.yml

```diff
@@ -0,0 +1,4 @@
+command:
+  # -i for interactive shell, so .bashrc is read
+  command: bash -i -c "pip \"\$@\"" --
+  in_service_with_role: streamlit
```

Comment:
Since these three commands are specific to the streamlit app and are not otherwise useful, please add them directly to the streamlit app.